exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/data_abalone.ipynb

```diff
@@ -106,21 +106,19 @@
    "outputs": [],
```

Comment:
<p>I learned in technical writing that contractions like <code>Let's</code> should be avoided. <a href="https://www.e-education.psu.edu/styleforstudents/c1_p15.html" target="_blank">https://www.e-education.psu.edu/styleforstudents/c1_p15.html</a> Maybe also check the remaining notebooks for these.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I don't think this rule is universally observed. I checked a few papers and books. Some use 'let's' some don't. But those who don't do not use 'let us' either, they just never needed this language construct.

Reply:
From my (very short) survey it looks like papers tend to phrase everything around "we", e.g. "we will present", "we are going to compute a gradient" etc. Books, on the other hand, use a more instructional format and that's where "let's" is used quite commonly.

Reply:
ok, I am convinced, others do this as well in notebooks
https://github.com/search?q=repo%3AAzure%2FMachineLearningNotebooks%20let%27s&type=code